src/earn/EarnInfoScreen.tsx

```diff
@@ -76,20 +75,13 @@ export default function EarnInfoScreen() {
         <View style={styles.detailsContainer}>
           <DetailsItem
             icon={<EarnCoins size={ICON_SIZE} color={Colors.black} />}
-            title={
-              isGasSubsidized
```

Comment:
we still support gas subsidy right? Are we no longer going to show that message on this screen?

Reply:
Correct, we are no longer showing the gas subsidy message.